packages/block-library/src/table/index.js

```diff
@@ -91,6 +91,8 @@ export const settings = {
 				},
 			],
 		},
+		defaultPadding: '1rem',
+		scale: 0.65,
```

Comment:
How do you calculate these values of the new props for each block? Why do we need them in some blocks only?

Reply:
baseline is `1` (actual size), and `0.65` means shrink to 65% of its original size.
Only few blocks such as `table`, `search` require this because their minimus size is way bigger than preview area.

Reply:
What about `defaultPadding`?

Reply:
it was to handle the space around the block. Reverted the api and set a default margin of 1rem in the CSS instead.